src/redux/book/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const bookSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      state.books.filter((item) => item.id === action.payload.id);
+    },
```

Comment:
- [ ] I am leaving the general comment here because I can not access the remove and submit buttons. However, I kindly suggest to make sure that it is possible to add and remove books. This is to ensure that you app is working as intended and meet the project requirements ☺️. You may consider using `useDispatch` as guided in the project requirements.